src/extensions/yfm/index.ts

```diff
@@ -4,10 +4,11 @@ export * from './Emoji';
 export * from './ImgSize';
 export * from './Monospace';
 export * from './Video';
-export * from './YfmCut';
 export * from './YfmConfigs';
+export * from './YfmCut';
 export * from './YfmFile';
 export * from './YfmHeading';
+export * from './YfmHtmlBlock';
```

Comment:
there is no need to export `YfmHtmlBlock` from package root, because in this case the package `@diplodoc/html-extension` should always be installed